src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/AppealSavedConfirmation.java

```diff
@@ -82,7 +77,7 @@ public PostSubmitCallbackResponse handle(
         String submitPaymentAppealUrl = "";
         String payOrSubmitLabel = "";
 
-        if (paymentOption != null && paymentOption.equals("payNow")) {
+        if ((paymentOption != null && (paymentOption.equals("payNow")) || appealType == AppealType.EA || appealType == AppealType.HU)) {
```

Comment:
Not sur eon the braces, looks like we have an additional pair, should simply be:
(paymentOption != null && (paymentOption.equals("payNow") || appealType == AppealType.EA || appealType == AppealType.HU))